common/protob/messages-cardano.proto

```diff
@@ -77,26 +79,6 @@ message CardanoSignTx {
     }
 }
 
-/**
- * Response: Serialised signed cardano transaction if tx_index is not specified.
- *              If tx_index is specified, trezor will wait for transaction
- * @next CardanoTxAck
- */
-message CardanoTxRequest {
-    optional uint32 tx_index = 1;   // index of requested transaction
-    optional bytes tx_hash = 2;     // hash of the signed transaction
-    optional bytes tx_body = 3;     // serialised body of the signed transaction
-}
-
-/**
- * Request: Reported transaction data
- * @next CardanoSignedTx
- * @next CardanoTxRequest
- */
-message CardanoTxAck {
-    optional bytes transaction = 1;
-}
-
 /**
  * Response: Serialised signed cardano transaction
  * @end
```

Comment:
as proposed offline, I agree the `tx_body` field should indeed be named `serialized_tx`  - that way it would be consistent with other coins and avoids confusion with "tx_body" as the structured payload of the transaction